tests/test_search_organisation_types.py

```diff
@@ -11,6 +11,7 @@ class TestSearchPostcode:
     endpoint = "organisationtypes"
     api_version = "1"
 
+    @pytest.mark.auth
     @pytest.mark.sandbox
     @pytest.mark.integration
     def test_search_organisation_types(self, get_api_key):
```

Comment:
we shouldn't add `auth` marker here. They are not testing authentication. Adding this marker will exclude them from sandbox which is wrong. Instead of excluding them, auth should be bypassed when env is sandbox. I'm not sure how this is possible with the new test-utils though.